packages/account-v2/src/pages/LoginHistory/LoginHistoryTable.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { Table } from '@deriv-com/ui';
+
+export const LoginHistory = () => {
+    // const { data, header } = useLoginHistoryTableData();
+
+    return (
+        <div className='flex flex-col '>
+            <Table
+                columns={[
+                    {
+                        header: 'Date and Time ',
+                    },
+                    {
+                        header: 'Action',
+                    },
+                    {
+                        header: 'Browser',
+                    },
+                    {
+                        header: 'IP Address',
+                    },
+                    {
+                        header: 'Status',
+                    },
+                ]}
+                data={[
+                    {
+                        action: 'Login',
+                        browser: 'Chrome v122.0.0.0',
+                        datetime: '2024-03-07 12:04:42 GMT',
+                        IpAddress: '942',
+                        status: 'Successful',
+                    },
+                ]}
+                isFetching={true}
+                renderHeader={header => <span>{header}</span>}
+                rowRender={data => (
```

Comment:
well `rowRender` is a bit weird name for a prop

Reply:
the prop rowRender is coming from deriv-com/ui so we cant actually change it